torch/nn/modules/loss.py

```diff
@@ -294,7 +294,8 @@ class KLDivLoss(_Loss):
 
     As with :class:`~torch.nn.NLLLoss`, the `input` given is expected to contain
     *log-probabilities* and is not restricted to a 2D Tensor.
-    The targets are given as *probabilities* (i.e. without taking the logarithm).
+    The targets are given as *probabilities* by default, but could be passed
```

Comment:
nit: this isn't really accurate (and I realize this was before your change) -- the targets are given however the users give them.  They are **interpreted** as probabilities by default.